flake.nix

```diff
@@ -110,10 +109,11 @@
         sops-nix = sops-nix.nixosModules.default;
       };
 
-    extendedNixosConfigurations =
-      mapAttrs
-      (_: config: nixosSystem {modules = [config ./dummy.nix] ++ attrValues extendedNixosModules;})
-      rawExamples;
+    mkNixosSystem = config: nixosSystem {modules = [config ./dummy.nix] ++ attrValues extendedNixosModules;};
```

Comment:
For another time: I'm not sure what this `dummy` configuration is for, it seems like it's a remnant of past times and can go away entirely.

Reply:
Can we please discuss this in a separate issue. The code is moved here, but it's out of scope for this PR.

Reply:
Deferring to #275